### PR TITLE
Calendar was created

### PR DESCRIPTION
**Pull request by illiashcherbak**
- [DEMO LINK](https://illiashcherbak.github.io/layout_calendar/)
- [TEST REPORT LINK](https://illiashcherbak.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--31">
+      <div class="calendar__cell calendar__cell--first-day--mon"></div>
```
Comment (KrisPlanka) [inline src/index.html]: move modifier to parent element

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+        display: block;
+      }
+    }
+
+    &:hover {
+      background-color: $color-cells-hover;
+      cursor: pointer;
```
Comment (KrisPlanka) [inline src/styles/blocks/calendar.scss]: remove cursor pointer from hover, and move to parent

**Review by KrisPlanka — CHANGES_REQUESTED** [review]
calendar should be in the middle of the screen (check readme.md)

**Review by KrisPlanka — APPROVED** [review]

**Review by ivan-kadykalo — APPROVED** [review]
Well done🚀

**Pull request closed without merging**